engines/gob/degob_script_v1.cpp

```diff
@@ -926,13 +926,72 @@ void Script_v1::o1_palLoad(FuncParams &params) {
 		break;
 
 	case 53:
+		skip(2);
+		return;
```

Comment:
Just making sure the return here is intentional, previous version had break here (and this is shared)